modules/ROOT/pages/private-cloud-edition-features.adoc

```diff
@@ -74,7 +74,7 @@ image::on-premises-features-3120e.png[]
 
 == Disclaimer
 
-By clicking the *Disclaimer* section you can set up a disclaimer message that your users see when they access the Anypoint Platform's log-in page.
+By clicking the *Disclaimer* section you can set up a disclaimer message that your users see when they access the Anypoint Platform's sign-in page.
```

Comment:
```suggestion
By clicking the *Disclaimer* section, you can set up a disclaimer message that your users see when they access the Anypoint Platform sign-in page.
```

Reply:
Should sign-in be *Sign in* in this case?

Reply:
Yes. I just did a S/R. So there might be a couple of these. I'll fix.